src/repairnator-jenkins-plugin/src/main/java/io/jenkins/plugins/main/RepairnatorJarDownloader.java

```diff
@@ -25,11 +26,15 @@ public String getLatestJarUrl(String snapshotUrl) throws IOException{
             latestJar = file.attr("href");
         }
 
-        return jarUrl + "/" +  latestJar;
+        System.out.println(doc);
+        System.out.println("LatestJarUrl: " + jarUrl  + latestJar);
+        return jarUrl +  latestJar;
     }
 
     public void downloadJar(String snapshotUrl) throws IOException{
-        String latestJarUrl = this.getLatestJarUrl(snapshotUrl);
+        /*String latestJarUrl = this.getLatestJarUrl(snapshotUrl);*/
+
+        String latestJarUrl = "https://repo.jenkins-ci.org/snapshots/fr/inria/repairnator/repairnator-pipeline/3.3-SNAPSHOT/repairnator-pipeline-3.3-20200316.204020-11-jar-with-dependencies.jar";
```

Comment:
hard-coded?

Reply:
Yes, there is a bug with the Jsoup that it only parse up to 9 pushed Jar on the page and I tried many solutions on the internet, but it does not seems to help so I decided to put it away for now.

Reply:
I see, what's the corresponding JSOUP bug report? Where the is JSOUP code?

Reply:
https://github.com/jhy/jsoup/issues/1259
https://stackoverflow.com/questions/21683019/jsoup-missing-content

Those are one of the solutions I tried but does not work. The most suspicious reason is that jsoup only scrape a snapshot of the page and we have to crawl for more. I would need to dig deep in the api for the correct config, so I just went with the fasted solution, just download the whole html page and parse it with Jsoup, since the repo page is usually not too big.